src/lib/probe.c

```diff
@@ -87,6 +175,14 @@ CbmDeviceProbe *cbm_probe_path(const char *path)
                 }
         }
 
+        /* Ensure that this parent disk really is LUKS */
+        if (blkid_probe_lookup_value(blk_probe, "TYPE", &value, NULL) != 0) {
```

Comment:
Well, we go to clean because there's nothing else we can do here, right? We failed to get the type, and our ret is NULL, so we end up just cleaning it up